vuefrontend/src/views/Home.vue

```diff
@@ -220,52 +226,18 @@ export default {
               for (var i = 0; i < this.resultData.length; i++) {
                 this.results.push(this.resultData[i]);
               }
-              console.log(this.resultData);
-              console.log("result array: ", this.results);
-              this.fetchingResult = false;
+              setTimeout(() => {
+                this.resetSearchButton();
```

Comment:
Use either `this.resetSearchButton()` or `this.fetchingResult = false`, and not switching between them to get some standard.